extensions/sql-database-projects/src/typings/vscode-mssql.d.ts

```diff
@@ -74,6 +74,10 @@ declare module 'vscode-mssql' {
          */
         getDatabaseNameFromTreeNode(node: ITreeNodeInfo): string;
 
+		/**
+         * Get the connection string for the provided connection ID
+       	 */
+		getConnectionString(connectionUri: string, includePassword: boolean): Promise<string>;
```

Comment:
This should be optional. When making updates here you should be copying over the file directly from the vs code mssql repo, which correctly has the parameter as optional. Did you do that here?